common/src/main/java/com/sequenceiq/cloudbreak/ccm/key/CcmResourceUtil.java

```diff
@@ -0,0 +1,29 @@
+package com.sequenceiq.cloudbreak.ccm.key;
+
+import java.util.Objects;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Provides utilities for dealing with resources.
+ */
+public class CcmResourceUtil {
+
+    private static final int CCM_KEY_ID_LENGTH = 36;
+
+    /**
+     * Private constructor to prevent instantiation.
+     */
+    private CcmResourceUtil() {
+    }
+
+    /**
+     * Returns the CCM SSH tunneling key ID for the specified resource CRN.
+     *
+     * @param resourceCrn the resource CRN
+     * @return the CCM SSH tunneling key ID for the specified resource CRN
+     */
+    public static String getKeyId(String resourceCrn) {
+        return StringUtils.right(Objects.requireNonNull(resourceCrn, "resourceCrn is null"), CCM_KEY_ID_LENGTH);
```

Comment:
This is just extracting the resource ID from the CRN, right? Consider just doing standard parsing.

```
return Crn.safeFromString(resourceCrn).getResource();
```

If not, it'd be helpful to add a comment explaining why 36 makes sense.